src/metabase/cmd.clj

```diff
@@ -208,7 +208,11 @@
   {:doc "Serialize Metabase instance into directory at `path`."
    :arg-spec [["-c" "--collection ID"            "Export only specified ID(s). Use commas to separate multiple IDs."
                :id        :collection-ids
-               :parse-fn  (fn [raw-string] (map parse-long (str/split raw-string #"\s*,\s*")))]
+               :parse-fn  (fn [raw-string] (->> (str/split raw-string #"\s*,\s*")
+                                                (map (fn [v]
+                                                       (if (re-matches #"\d+" v)
```

Comment:
It's technically possible that the entity id is all digits, right? It's a pity we didn't put some identifier prefix on them.

Reply:
Alternatively we force the input to add its own prefix, like `eid:03A42F8B7`

Reply:
Uhhh I thought about that a bit and was like "maybe let it break, no big deal" 🤣

Let me think about it a bit, I don't like too much cross-checking... Also it's the same problem with the HTTP API, just less explicit, because it receives arguments from query string.